config/config.json

```diff
@@ -1,7 +1,7 @@
 {
   "appid": "",
   "group": "",
-  "webscene": "19faa71a3bf6468cae35b4fce9393a7d",
+  "webscene": "",
```

Comment:
(Overlapping comments) @kellyhutchins:  If no url param is supplied, one sees the error page with "main:: Could not load an item to display"